src/Renci.SshNet/Security/Cryptography/RsaDigitalSignature.cs

```diff
@@ -41,13 +41,20 @@ public RsaDigitalSignature(RsaKey rsaKey, HashAlgorithmName hashAlgorithmName)
         /// </returns>
         protected override byte[] Hash(byte[] input)
         {
-            return _hash.ComputeHash(input);
+#if !NET462
+            using var hash = IncrementalHash.CreateHash(_hashAlgorithmName);
+            hash.AppendData(input);
+            return hash.GetHashAndReset();
+#else
+            using var hash = CryptoConfig.CreateFromName(_hashAlgorithmName.Name) as HashAlgorithm
```

Comment:
`CryptoConfig` is a somewhat legacy API, and `CreateFromName` is annotated with `[RequiresUnreferencedCode]`. I'm not sure which of these points is the root cause for https://github.com/sshnet/SSH.NET/issues/859#issuecomment-1764623253 (needs more details), but either way we can just use `IncrementalHash` instead.